### ISSUE TITLE
TypeLITE [TsClass], [TsInterface] and [TsIgnore] attributes not picked up

### ISSUE THREAD

**Issue by vmandic** [slovely/TypeScriptDefinitionsGenerator#13]
So when using the tool **only the classes and interfaces filtered by the namespace filter** get picked up for translation. It would be handy if the generator could recognize and respect the TypeLITE's attributes [TsClass], [TsIgnore] and [TsInterface] including the filter.

**Comment by slovely** [comment]
I think the attributes only work if you reference the exact same version of TypeLITE as the generator references - which is currently a pretty old v1.8.1.  I can't check this right now, but will try to confirm soon; unless it's easy for you to change the version of TypeLITE you are referencing?

**Comment by vmandic** [comment]
Yes, you are correct, the line below specifies what you have said: https://github.com/slovely/TypeScriptDefinitionsGenerator/blob/master/src/TypeScriptDefinitionsGenerator/packages.config#L12

...and I confirm the explanation you have provided, i.e. changing the .Lib version worked.

**Comment by slovely** [comment]
I'm not entirely show that a fix for this will be easy... the attribute matching is in the TypeLITE library itself.  I think I'll probably end up just updating the internal version and document that it needs to match.  Otherwise it would likely involve having a property in the '.props' file to point to the users version of TyepLITE, and dynamically loading and using that one.
